.eslintrc

```diff
@@ -0,0 +1,69 @@
+{
+  "extends": [
+    "eslint:recommended",
+    "plugin:import/errors",
+    "plugin:import/warnings"
+  ],
+  "plugins": [
+    "react"
+  ],
+  "parserOptions": {
+    "ecmaVersion": 6,
+    "sourceType": "module",
+    "ecmaFeatures": {
+      "jsx": true
+    }
+  },
+  "env": {
+    "es6": true,
+    "browser": true,
+    "node": true,
+    "jquery": true,
+    "mocha": true
+  },
+  "rules": {
+    "quotes": 0,
```

Comment:
0 means no alerts
1 means warning
2 means errors. 

I'd like to keep this semantic, firstable for the file weight `"off"` is 4 bytes `0` is one, and then for the community as I never saw an "off" online, moreover, any programmer knows that `0 == false` or in our case silenced rule 😉